src/cal-heatmap.js

```diff
@@ -1868,7 +1871,7 @@ CalHeatMap.prototype = {
 		if (this.options.highlight.length > 0) {
 			for (var i in this.options.highlight) {
 				if (this.options.highlight[i] instanceof Date && this.dateIsEqual(this.options.highlight[i], d)) {
-					return " highlight" + (this.isNow(this.options.highlight[i]) ? " now": "");
+          return this.isNow(this.options.highlight[i]) ? " highlight-now": " highlight";
```

Comment:
I think you should always add the `highlight` class, since some people will not want to highlight the current time differently